src/tests/ut/calendar/calendar_test.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "calendar.hpp"
+#include "clinic_facade.hpp"
+#include "day.hpp"
+#include "day_name.hpp"
+#include "month.hpp"
+#include "year.hpp"
+#include "gtest/gtest.h"
+#include <chrono>
+
+class CalendarTestFixture : public ::testing::Test
+{
+  public:
+    std::shared_ptr<Calendar> cal;
+
+  protected:
+    void SetUp() override
+    {
+        Calendar::createCalendar();
```

Comment:
Here we can see one problem of having statiic Clinic.
In production code we will have to create the calendar explicitly, most likely in main().

If Clinc where created in proper place in production code as non-static in nature object, it could create it's calendar in the constructor behind the curtain.